src/components/Dashboard/TodoList.vue

```diff
@@ -515,6 +515,7 @@ export default class TodoList extends Vue {
   @Getter getTasks!: Array<ApiTaskIF>
   @Getter getTodoListResource!: TodoListResourceIF
   @Getter isBenBcCccUlc!: boolean
+  @Getter isGoodStanding!: boolean
```

Comment:
This was being inherited from a mixin, but I think it's better to directly import it in each file it's used (otherwise you lose visibility of which component uses which getter).